lib/controllers/page_controller.rb

```diff
@@ -17,7 +17,11 @@ def restrict_concealed(page)
 
   get '/' do
     @page = Page.order(:id).first
-    redirect "new" unless @page
+    if logged_in?
```

Comment:
Hmm, if you're not logged in and the page exist? Won't this always show a new page then?

Nonetheless I think we need to tweak the logics here for readability.

Reply:
> if you're not logged in and the page exist, won't this always show a new page then?

I think you are right about this

I had hoped we could add "logged in" tests at a later point, but sounds like we need them now, in this PR

FYI, it might be a while before I can get back to this PR, but I don't have problem leaving it open until that

Reply:
Yeah, no hurry. Usually we have something in the database ... :)

Reply:
Maybe something like this would be easier to reason about?

```Ruby
  get '/' do
    @page = Page.order(:id).first || Page.new(title: "Förstasidan")

    if @page.new? && logged_in?
      redirect "new"
    end

    @page_title = @page.title
    haml :show
  end
```

_Not tested_

Reply:
> if you're not logged in and the page exist

I think that is covered by our existing tests

https://github.com/Starkast/wikimum/blob/25cfc613e6b642a7881a95fe5a3985324c6a06a0/test/integration/app_test.rb#L21-L23

https://github.com/Starkast/wikimum/blob/25cfc613e6b642a7881a95fe5a3985324c6a06a0/test/integration/app_test.rb#L29-L32